app/templates/components/questionnaire/job-name.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="row">
+    <div class="col-md-6">
```

Comment:
`col-md-6` limits this to half the width. Was that intentional or just a holdover from the component this was based on?